go.mod

```diff
@@ -5,6 +5,7 @@ go 1.13
 require (
 	github.com/go-logr/logr v0.1.0
 	github.com/golang/protobuf v1.4.2 // indirect
+	github.com/google/uuid v1.1.1
```

Comment:
We also need vendored deps update. `make tidy`.

Reply:
It already exists as a transitive dependency https://github.com/storageos/api-manager/tree/master/vendor/github.com/google/uuid .